### PR TITLE
Use a fade animation on the header background color change

### PR DESCRIPTION
**Pull request by cjb**
@keybase/react-hackers @keybase/design What do you think?

pinentry

### CODE REVIEW

File: shared/tracker/header.render.desktop.js [hunk 1]
```diff
@@ -106,7 +106,8 @@ const styles2 = {
     top: 0,
     ...globalStyles.flexBoxRow,
     height: 90,
-    width: 320
+    width: 320,
+    transition: 'background 0.2s linear'
```
Comment (chrisnojima) [inline shared/tracker/header.render.desktop.js]: maybe we want to move this into style-guide along w/ the other transition (we could also just try it)

**Comment by cecileboucheron** [comment]
I like it!

File: shared/tracker/header.render.desktop.js (hunk 1 above)
Comment (cjb) [inline shared/tracker/header.render.desktop.js]: thanks, moved it

**Comment by chrisnojima** [comment]
:+1:

**Comment by cbostrander** [comment]
Looks slick Chris! Thanks for sending along.

**Comment by malgorithms** [comment]
cooooool

**Pull request merged**